src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -819,21 +741,31 @@ configure(struct sched_cmd *cmd)
 		exec->configured = 1;
 	}
 
-	exec->cu = vzalloc(sizeof(struct zocl_cu) * exec->num_cus);
+	exec->zcu = vzalloc(sizeof(struct zocl_cu_new) * exec->num_cus);
+	if (!exec->zcu) {
+		DRM_ERROR("Cound not allocate CU objects\n");
+		return -ENOMEM;
+	}
 
 	for (i = 0; i < exec->num_cus; i++) {
+		if (cfg->data[i] & ~ZOCL_KDS_MASK == ACCEL_ADAPTER) {
+			/* If the ACCEL adapter is used */
+			acc_cu = 1;
```

Comment:
should't we clear acc_cu to zero someware inside the for loop?

Reply:
fixed